<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RobertsCrossClass</class>
 <widget class="QMainWindow" name="RobertsCrossClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RobertsCross</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="input">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>Wejście</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="output">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>Wynik</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QTextBrowser" name="results">
        <property name="openLinks">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Wątki:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="threads">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>64</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Biblioteka</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="cppRadio">
            <property name="text">
             <string>C++</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="asmRadio">
            <property name="text">
             <string>Asembler</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Wybierz obraz</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="computeButton">
          <property name="text">
           <string>Oblicz</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>881</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RobertsCross.qrc"/>
 </resources>
 <connections/>
</ui>
